compiler/tests/fail/stack_allocation/x86-64/regions_not_equal_array.jazz

```diff
@@ -0,0 +1,12 @@
+export fn main (reg u64 i) -> reg u64 {
+  stack u64[2] s;
+  reg ptr u64[2] r;
+  reg u64 res;
+
+  s[0] = 0;
+  r = s[i:2];
```

Comment:
zone = [0:16][(8 * ((int /* of u64 */) i.222)):16] }
I think that "/* of u64 */" make the message hard to read.

Reply:
Also the fact that we use long name

Reply:
I fixed the long name (not shown if `-debug` is not passed). For the cast, cf. https://github.com/jasmin-lang/jasmin/pull/1124.